index.bs

```diff
@@ -1131,7 +1131,7 @@ A <dfn>reflective module record</dfn> is a kind of module record. In addition to
   <tr>
     <td>\[[LocalExports]]</td>
     <td>A List of Strings</td>
-    <td>The set of exported names stored in this module's environment.</td>
+    <td>The set of exported names accessible in this module's environment.</td>
```

Comment:
it is not longer stored directly in the env record, but it is accessible from there.